data/noiseprofiles.schema

```diff
@@ -20,7 +20,7 @@
             "type": "string",
             "pattern": "^[a-zA-Z ]+$",
             "minLength": 3,
-            "maxLength": 13,
+            "maxLength": 14,
```

Comment:
Could we get rid of those limits, as we already enforce to use one of the valid enum values further down?